src/assets/drizzle/styles/components/_command.scss

```diff
@@ -10,3 +10,8 @@
 .#{$app-namespace}-c-Command code {
   font-size: 0.9rem;
 }
+
+.#{$app-namespace}-c-Command--inline {
+  padding: 0 0.25rem;
+  font-size: 0.9rem;
```

Comment:
BEM modified classes utilize all the classes of the parent. So, this should be structured so that the element gets all of the original c-Command styles. That would make the font-size declaration in c-Command--inline not needed.

Reply:
c-Command doesn't actually have a font-size. There is a "c-Command code" selector that has the same font size, but that selector doesn't apply to the places where I'm using c-Command--inline. 

I didn't want to mess with the "c-Command code" selector because I didn't want to have to rewrite all of the places where it's used, but I can do that if it's the right thing to do.

Reply:
I think we should revisit it because it’s weird the way we have it setup now. Shouldn’t be selecting elements directly. I’d like to know why we have the c-Command code instead of a modified class.

Reply:
I wonder if the `<code>` element is being generated or something, making it hard to put a class directly on it. I remember being really confused on how to use that class properly, so I support revisiting.

Reply:
I'll revisit all the places where we're using <code>.  We also have a 'drizzle-CodeHighlight' class on a bunch of code elements that doesn't appear to be used in any CSS selectors.